CreateGroup/function.json

```
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": ["post"],
+      "route": "groups"
```

```suggestion
      "route": "group/"
```
read about rest API naming
https://stackoverflow.com/questions/40223421/api-routes-naming-convention